src/app/Caches/CacheMetrics.tsx

```diff
@@ -186,6 +195,11 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
             />
           </div>
         </CardBody>
+        <CardFooter>
+        <ExpandableSection toggleTextExpanded="Show less" toggleTextCollapsed="Interpreting data access statistics">
+        Data access statistics show info about cache operations to access data.
+        </ExpandableSection>
+        </CardFooter>
```

Comment:
@karesti Hey, what do you think about adding an expandable footer to the cache metrics donut graph? I'm not sure about how to add tooltips for every data access statistic. Also it seems more compact to add a lot of text to an expandable section rather than a lot of tooltips that might clutter the UI. Just looking for any suggestion on how best to do this. Thanks.